docs/resources/storage.md

```diff
@@ -150,13 +129,6 @@ Required:
 - **retention** (Number) The number of days before a backup is automatically deleted
 - **time** (String) The time of day when the backup is created
 
-Notes:
```

Comment:
Is there some way of adding such notes to the output? Via comment or something? If not, then we should move this information to examples somehow I guess

Reply:
Missed those notes :) I created new template for server and added those notes to template.